github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1032,59 +892,32 @@ periodics:
     preset-docker-mirror-proxy: "true"
     preset-kubevirtci-quay-credential: "true"
     preset-shared-images: "true"
+  max_concurrency: 1
+  name: periodic-kubevirt-performance-cluster-density-test
   reporter_config:
     slack:
       job_states_to_report: []
-  name: periodic-kubevirt-performance-cluster-density-test
-  max_concurrency: 1
   spec:
     containers:
     - command:
       - /usr/local/bin/runner.sh
       - /bin/sh
       - -c
-      - |
-        # install yq
-        curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
-        chmod +x ./yq && mv ./yq /usr/local/bin/yq
-
-        # get kubectl
-        curl -L "https://dl.k8s.io/release/v1.19.7/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl
-        chmod +x /usr/local/bin/kubectl
-
-        # get kubeconfig
-        source ../project-infra/hack/manage-secrets.sh
-        decrypt_secrets
-        extract_secret 'kubeconfigPerf' /kubeconfig
-
-        # login quay.io
-        cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
-
-        # run the test
-        make clean
-        make bazel-build
-        make cluster-sync
-        make cluster-deploy
-        start_timestamp=$(date -u +%Y-%m-%dT%TZ
-        sleep 30
-        _out/cmd/perfscale-load-generator/perfscale-load-generator \
-          -container-prefix $DOCKER_PREFIX \
-          -container-tag $DOCKER_TAG \
-          -v 6 \
-          -workload tools/perfscale-load-generator/examples/workload/kubevirt-density/kubevirt-density.yaml
-
-        # collect metrics      
-        kubectl -n monitoring port-forward service/prometheus-stack-kube-prom-prometheus ${PROMETHEUS_PORT} &> /dev/null &
-        sleep 30
-        stop_timestamp=$(date -u +%Y-%m-%dT%TZ)
-        cat <<EOF >perfscale-audit-cfg.json
-        {
-          "prometheusURL": "http://127.0.0.1:${PROMETHEUS_PORT}",
-          "startTime": "$start_timestamp",
-          "endTime": "$stop_timestamp"
-        }
-        EOF
-        _out/cmd/perfscale-audit/perfscale-audit -config-file perfscale-audit-cfg.json
+      - "# install yq\ncurl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64\nchmod
```

Comment:
I wonder if we can keep the inline formatting somehow. Although such long scripts should not be really part of the job definition.

Reply:
@dhiller FYI

Reply:
Maybe some setting is available at the Marshaller, I'll take a look.